src/main/kotlin/org/jetbrains/plugins/bsp/magicmetamodel/impl/workspacemodel/impl/updaters/transformers/ModuleDetailsToGoModuleTransformer.kt

```diff
@@ -11,48 +11,51 @@ import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.GoModuleDepe
 import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.ModuleDetails
 import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.toBsp4JTargetIdentifier
 import java.net.URI
-import java.nio.file.Path
 import kotlin.io.path.toPath
 
 internal class ModuleDetailsToGoModuleTransformer(
-    private val targetsMap: Map<BuildTargetId, BuildTargetInfo>,
-    private val projectDetails: ProjectDetails,
-    moduleNameProvider: ModuleNameProvider,
-    projectBasePath: Path,
+  private val targetsMap: Map<BuildTargetId, BuildTargetInfo>,
+  private val projectDetails: ProjectDetails,
+  moduleNameProvider: ModuleNameProvider,
 ) : ModuleDetailsToModuleTransformer<GoModule>(targetsMap, moduleNameProvider) {
-    override val type = "GO_MODULE"
+  override val type = "GO_MODULE"
 
-    override fun transform(inputEntity: ModuleDetails): GoModule {
-        val goBuildInfo = extractGoBuildTarget(inputEntity.target) ?: error("extract nie działa")
+  override fun transform(inputEntity: ModuleDetails): GoModule {
+    val goBuildInfo = extractGoBuildTarget(inputEntity.target) ?: error("Transform error, cannot extract GoBuildTarget")
 
-        return GoModule(
-            module = toGenericModuleInfo(inputEntity),
-            importPath = goBuildInfo.importPath ?: error("cos nie dziala"),
-            root = URI.create(inputEntity.target.baseDirectory).toPath(),
-            goDependencies = inputEntity.moduleDependencies.mapNotNull { targetsMap[it] }.map { buildTargetInfo ->
-                val buildTarget = projectDetails.targets.find { it.id == buildTargetInfo.id.toBsp4JTargetIdentifier() }
-                    ?: error("find nie działa")
-                val goBuildInfoo = extractGoBuildTarget(buildTarget) ?: error("extract nie działa")
-                GoModuleDependency(
-                    importPath = goBuildInfoo.importPath ?: error("nie ma importPatha"),
-                    root = URI.create(buildTarget.baseDirectory).toPath()
-                )
-            }
-        )
-    }
+    // TODO: replace importPath: String? in GoBuildTarget since Vgo modules require String
+    return GoModule(
+      module = toGenericModuleInfo(inputEntity),
+      importPath = goBuildInfo.importPath ?: "No importPath for GoModule",
```

Comment:
```suggestion
      importPath = extractGoBuildTarget(inputEntity.target)?.importPath ?: "No importPath for GoModule",
```

Reply:
I'm leaning towards a change so that the importpath part doesn't throw any errors

Reply:
To juz jak Ci wygodnie, dla mnie oba pewnie git, w tym transformerze Javy i Pythona jest od razu tworzony modul, bez sprawdzania czy to java czy tam python czy nie

Reply:
wiem, różnica jest taka że u nich nie używają tej funkcji extract, więc zły target chyba nie psuje im tak dużo jak nam